src/PublishTesRunnerToTesApiArtifacts/PublishTesRunnerToTesApiArtifacts.csproj

```diff
@@ -0,0 +1,135 @@
+        <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
+        <OutputItemType>None</OutputItemType>
+        <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+        <Private>False</Private>
+    </ProjectReference>
+    <None Condition=" '$(TesApiWebArchive)' != '' "
+          Include="$(TesApiWebArchive)\tesapi.dll"
+          MsBuildSourceProjectFile="$(TesApiWebArchive)\proj"
+          OutputPath="tesapi.dll"
+          TargetPath="tesapi.dll" />
+  </ItemGroup>
+
+  <!--<Target Name="__AssignPublishProperties" BeforeTargets="ResolveProjectReferences" AfterTargets="PrepareProjectReferences">
+    <ConvertToAbsolutePath Paths="$(OutputPath)scripts\">
+      <Output TaskParameter="AbsolutePaths" PropertyName="__TesRunnerPublishDir"/>
+    </ConvertToAbsolutePath>
+
+    <ItemGroup>
+      <__ProjectNotReferenced Include="@(_MSBuildProjectReferenceExistent)" />
+      <_MSBuildProjectReferenceExistent Remove="@(__ProjectNotReferenced)" />
+      <_MSBuildProjectReferenceExistent Include="@(__ProjectNotReferenced)">
+        <Properties>%(__ProjectNotReferenced.AdditionalProperties);PublishDir=$(__TesRunnerPublishDir);%(__ProjectNotReferenced.SetConfiguration);%(__ProjectNotReferenced.SetPlatform);%(__ProjectNotReferenced.SetTargetFramework)</Properties>
+        <AdditionalProperties></AdditionalProperties>
+        <GlobalPropertiesToRemove>%(__ProjectNotReferenced.GlobalPropertiesToRemove);PublishSingleFile;PublishTrimmed;SelfContained</GlobalPropertiesToRemove>
+      </_MSBuildProjectReferenceExistent>
+    </ItemGroup>
+  </Target>-->
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets"/>
+
+  <UsingTask TaskName="ConvertNoneToPublishDir" TaskFactory="RoslynCodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll">
+    <ParameterGroup>
+      <PublishedFiles ParameterType="Microsoft.Build.Framework.ITaskItem[]" Required="true" />
+      <PublishedDirectories ParameterType="Microsoft.Build.Framework.ITaskItem[]" Output="true" />
+    </ParameterGroup>
+    <Task>
+      <!--<Reference Include="System.Linq"/>-->
+      <Using Namespace="System.Linq" />
+      <Using Namespace="System.IO" />
+      <Using Namespace="Microsoft.Build.Framework" />
+      <Using Namespace="Microsoft.Build.Utilities" />
+      <Code Type="Fragment" Language="cs">
+        <![CDATA[
+            var dirs = new List<ITaskItem>();
+            foreach (var file in PublishedFiles)
+            {
+                Log.LogMessage(MessageImportance.Normal, "Parsing {0}", file.ItemSpec);
+                var dirInfo = Directory.GetParent(Path.Combine(Directory.GetParent(file.GetMetadata("MsBuildSourceProjectFile")).FullName, file.GetMetadata("OutputPath")));
+                var dir = new TaskItem(dirInfo.FullName);
+                file.CopyMetadataTo(dir);
+                dirs.Add(dir);
+            }
+            //PublishedDirectories = Enumerable.DistinctBy(dirs, item => item.ItemSpec).ToArray();
```

Comment:
Same as above, commented out code should be removed?

Reply:
Done